udata/harvest/actions.py

```diff
@@ -153,28 +154,33 @@ def preview(ident):
     return backend.harvest()
 
 
-def schedule(ident, minute='*', hour='*',
+def schedule(ident, cron=None, minute='*', hour='*',
```

Comment:
Do we still need the other karkgs? `minute`, `hour`...

Reply:
Yes, we need it, `udata harvest schedule` command line use it.